diff --git a/dist/log4brains b/dist/log4brains
index 2a7ad52bb2bf14d0e11fd1630821243c9b2dff80..ee973687ef3591a3642f1054618f11af801ec375 100755
--- a/dist/log4brains
+++ b/dist/log4brains
@@ -6,11 +6,11 @@ const { spawn } = require("child_process");
 // that enables us to force the --openssl-legacy-provider NODE_OPTIONS flag without asking the user to do it himself.
 // This is really hacky but will be removed  as soon as we manage to upgrade our dependencies.
 const child = spawn(
-  "node",
-  [path.join(__dirname, "index.js"), ...process.argv.slice(2)],
+  "yarn",
+  ["node", path.join(__dirname, "index.js"), ...process.argv.slice(2)],
   {
     env: { ...process.env, NODE_OPTIONS: "--openssl-legacy-provider" },
     cwd: process.cwd(),
-    stdio: "inherit"
+    stdio: "inherit",
   }
 );
